manager/src/main/java/com/redhat/service/smartevents/manager/models/Error.java

```diff
@@ -0,0 +1,84 @@
+package com.redhat.service.smartevents.manager.models;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import org.hibernate.annotations.Type;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+import io.quarkiverse.hibernate.types.json.JsonTypes;
+
+@NamedQueries({
+        @NamedQuery(name = "ERROR.findByBridgeIdOrdered",
+                query = "from Error where bridge_id=:bridgeId order by id desc")
+})
+@Entity
+@Table(name = "ERROR")
+public class Error {
+
+    @Id
+    @SequenceGenerator(name = "errorId", sequenceName = "ERROR_ID_SEQUENCE", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "errorId")
+    private Long id;
+
+    @Column(name = "bridge_id", nullable = false)
```

Comment:
Perhaps all columns could be `updatable = false` too?

Reply:
Good catch! Done :+1: